lib/utility/constants.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+import 'package:acnhpedia/view/search_screen.dart';
+
+InputDecoration textFieldDecor = InputDecoration(
+  filled: true,
+  fillColor: Color(0xffbee893),
+  hintText: 'Enter an ACNH character',
+  border: OutlineInputBorder(
+      borderSide: BorderSide.none, borderRadius: BorderRadius.circular(20)),
+);
```

Comment:
move this to components or widgets directory, your constants should have static/constant variables, not generic widgets